src/libs/actions/Report.js

```diff
@@ -1040,7 +1041,17 @@ function addAction(reportID, text, file) {
         // the same way report actions can.
         persist: !isAttachment,
     })
-        .then(({reportAction}) => updateReportWithNewAction(reportID, reportAction));
+        .then((response) => {
+            if (response.jsonCode === 408) {
+                Growl.show('Upload Failed. File is not supported.', CONST.GROWL.ERROR);
```

Comment:
@iwiznia We should show show the localized string here. So I am pinging you to ask what is the best approach, there is not component involved and thus HOC can't be used.
What I think is that we can subscribe to the `prefferedLocale` using Onyx.Connect and then call the translate directly.

Reply:
Yeah, that's the way to go.